Consolidate tree-sitter grammars on ast-grep-language + inline ast-grep-core fallback (#487)
usercise/spelunk: docs/commands.md

`semantic`/`hybrid` uses LinearRAG: a two-stage entity-activation + personalised
PageRank pipeline that improves multi-hop recall over raw KNN. `text` and
`ast-grep` need no embedding model or server. Structural (`ast-grep`) search
runs **in-process** — it is compiled into the `spelunk` binary, so no external

> `ast-grep` tool needs to be installed.

Author: ```suggestion
`ast-grep` need no embedding model or server.
```

Keep as it was.

Author: Applied verbatim in dfa6f451. Thanks.

| `--format text\|json\|jsonl` | text | Output format |
| `-d, --db <path>` | auto | Override database path |
| `--no-stale-check` | false | Suppress the stale-index warning |

> | `--live` | false | Skip the index and scan live files with in-process structural matching (no external tool required) |

Author: ```suggestion
| `--live` | false | Skip the index and scan live files directly |
```

Author: Applied verbatim in dfa6f451. Thanks.
